### PR TITLE
move Upgrade to Monitoring docs

### PR DESCRIPTION
**Pull request by lauranovich**
Mirror the content until deleted on the source

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -95,6 +95,7 @@ def setup(sphinx):
 
 rst_prolog = """
 .. |version| replace:: 3.7.0
+.. |mon_root| replace::  `Scylla Monitoring Stack <https://monitoring.docs.scylladb.com>`__
```
Comment (tzach) [inline docs/source/conf.py]: Why not link to `/` ?

Comment (lauranovich) [reply docs/source/conf.py]: true - I'll send the correction

Comment (tzach) [reply docs/source/conf.py]: Please validate it works

**Comment by lauranovich** [comment]
I amended the link. It works in regular preview. I am not sure how it will work in multiversion.

**Comment by tzach** [comment]
LGTM

**Comment by lauranovich** [comment]
can we please backport this to 3.8?

**Comment by tzach** [comment]
@lauranovich why is this urgent  backport?
Most updates in doc (and software) should land on upstream.
Backport is for urgent fixes only, which is not the case here

**Comment by lauranovich** [comment]
@amnonh when can I expect to see this merge in a branch? It should have made it to 3.8, no?

**Comment by amnonh** [comment]
See the previous conversation you had with Tzach, it didn't make it to 3.8 and wasn't backport, I'm working to branch 3.9, so probably we'll have a RC this week

**Pull request merged**